core/unit_test/TestReduce.hpp

```diff
@@ -510,12 +512,13 @@ TEST(TEST_CATEGORY, int_combined_reduce_mixed) {
 
   auto result3_v = Kokkos::View<int64_t, Kokkos::HostSpace>{"result3_v"};
 
-  Kokkos::parallel_reduce(nw, functor_type(nw), result1_v, result2,
+  Kokkos::parallel_reduce("int_combined-reduce_mixed",
+                          Kokkos::RangePolicy<TEST_EXECSPACE>(0, nw),
+                          functor_type(nw), result1_v, result2,
                           Kokkos::Sum<int64_t, Kokkos::HostSpace>{result3_v});
```

Comment:
Is this a driveby change on combined reducers? If so, should it be in a separate pull request?

Reply:
its not necessarily a drive-by change. It is a change specifically necessary for OpenMPTarget since OpenMPTarget doesn't have something like UVM or HostPinned memory which was used previously for these tests.